pkg/util/encryption/xchacha20poly1305.go

```diff
@@ -53,3 +53,7 @@ func (c *xChaCha20Poly1305) Seal(input []byte) ([]byte, error) {
 
 	return append(nonce, c.aead.Seal(nil, nonce, input, nil)...), nil
 }
+
+func (c *xChaCha20Poly1305) SealSecretVersion() string {
+	return ""
```

Comment:
This is my first glance at the encryption package, why does this return `""`?

Reply:
@SrinivasAtmakuri This probably needs a comment. I'm guessing that it's blank because it will never have a secret version?

Reply:
So this puzzles me too.